### DISCUSSION TITLE
lotus-miner storage attach --init --seal ERROR: could not get API info for StorageMiner: repo directory does not exist. Make sure your configuration is correct

### DISCUSSION THREAD

**Discussion by tutorialevideo** [filecoin-project/lotus#7913] [Storage Provider] [answered]
Anyone can help with this error?

lotus-miner storage attach --init --seal
ERROR: could not get API info for StorageMiner: repo directory does not exist. Make sure your configuration is correct

**Comment by de-authority** [comment]
It doesn't look like a bug. you can ask for help here in slack：https://filecoinproject.slack.com/archives/CEGN061C5

**Comment by Reiers** [comment]
**Hi @tutorialevideo** 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to [Lotus Discussions](https://github.com/filecoin-project/lotus/discussions) for help and troubleshooting.

**Comment by LexLuthr** [comment, marked as answer]
@tutorialevideo it looks like either storage was not initialised or is missing. Can you please check if you have run through all the pre-requisites correctly?

https://lotu.sh/docs/storage-providers/setup/

**Reply by LexLuthr** [reply]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
